astro/src/content/docs/release-notes/index.mdx

```diff
@@ -45,6 +45,50 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
 
+<ReleaseNoteHeading version='1.57.0' releaseDate='April 4, 2025' name='Webhook Wombat'/>
+
+<GeneralMigrationWarning>
+  This release provides a handful of improvements to the webhook event log. In versions 1.53.0 through 1.56.0, very large webhook event log volumes could impact performance during log cleaning operations. 
+
+  If you are currently using a version of FusionAuth earlier than 1.53.0, we recommend upgrading to version 1.57.0 and not an intermediate version.
+</GeneralMigrationWarning>
+
+<DatabaseMigrationWarning />
+
+### Changed
+* <RelatedToIssue issue="3037">
+    The process for applying webhook event log retention rules, and cleaning out old webhook event logs, was formerly turned off by default, causing all webhook event log entries to be retained. This will now be defaulted to `on` in new FusionAuth installations. Existing installations that are being upgraded will retain their former setting.
+  </RelatedToIssue>
+
+### Fixed
+* <Issue issue="3026">
+    The TypeScript client is missing support for patching the `Entity.data` field. Support for this endpoint has been added.
+  </Issue>
+
+* <Issue issue="3027">
+    Some client libraries are missing support for certain CRUDP operations. This fix adds support for these, and also adds additional automated checks to verify that client libraries have full CRUDP support domain objects in their associated APIs.
+  </Issue>
+
+### Enhancements
+* <Issue issue="2980">
+    When prompted for how you heard about FusionAuth in the setup wizard, you may now supply "ChatGPT/LLMs/AI" as a standard response.
+  </Issue>
+
+* <Issue issue="3037">
+    Two enhancements were added to the configuration for controlling webhook event logging behavior:
+    * Webhook event logging can now be turned off. Previously, you were only able to reduce the retention period to a very small window, however in this case the logging and log management would still occur.
+    * Webhook event log retention is now enabled by default. Previously, it was disabled by default.
+  </Issue>
+
+* <Issue issue="3038">
+    Webhook event log searching was improved by setting a default 1-hour search window when an unbounded search is made. Webhook event logging can create a large number of log records, and the performance of unbounded searches scaled roughly linearly with the number of log records, leading to less performant searches as the data size increased.
+  </Issue>
+
+* <Issue issue="3039">
+    The process that applies webhook event log retention rules is getting some performance improvements. This is a background process that generally does not impact performance, but could consume a disproportionate amount of system resources when under heavy load.
```

Comment:
The second part of the sentence is not an independent clause, so it is not a compound sentence.
```suggestion
    The process that applies webhook event log retention rules is getting some performance improvements. This is a background process that generally does not impact performance but could consume a disproportionate amount of system resources when under heavy load.
```